DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -37,11 +37,20 @@ public class EventsSheetsService : IEventsSheetsService
         private readonly SheetsService sheetsService;
         private int largestKey;
 
-        private const string KeyColumn = "A";
-        private const string NameColumn = "B";
+        private const string KeyColumnLetter = "A";
+        private const int KeyColumnIndex = 0;
+
+        private const string NameColumnLetter = "B";
+        private const int NameColumnIndex = 1;
+
         private const string DescriptionColumn = "C";
-        private const string TimeColumn = "D";
-        private const string LocationColumn = "E";
+        private const int DescriptionIndex = 2;
+
+        private const string TimeColumnLetter = "D";
+        private const int TimeColumnIndex = 3;
+
+        private const string LocationColumnLetter = "E";
+        private const int LocationColumnIndex = 4;
```

Comment:
I've merged the pairs of `*ColumnLetter` and `*ColumnIndex` together in be4ad67 🙂 Should I also look to merge the list of these pairs into a `MetadataSheet` object? It would be nice to make that logical grouping explicit, but it would make referring to the columns a bit more verbose so I don't know which is best

Reply:
It wasn't quite what I meant (I was thinking of just a static class with all the public const strings) but I like your solution 🙂 I think it's fine to leave as it is now, it's only five variables so a lot more manageable 🙂

Reply:
Ah I see 🙂 Well, at least it's better now